python/generate/templates/api.mustache

```diff
@@ -13,7 +13,10 @@ from {{packageName}}.api_client import ApiClient
 from {{packageName}}.exceptions import (  # noqa: F401
     ApiTypeError,
     ApiValueError
-)
+){{#hasImport}}
+
```

Comment:
Is the purpose of this to create a blank line between these imports and those above? These imports are of the same group as the existing LUSID imports brought in above and according to PEP8 they should be grouped together with no blank line in between. https://www.python.org/dev/peps/pep-0008/#imports

I suggest removing the `{{#hasImport}}` and `{{/hasImport}}` tags to comply with PEP8 and simplify the template change.

Reply:
Yes indeed, it was an erroneous stylistic choice!